meshroom/core/graph.py

```diff
@@ -1208,6 +1208,7 @@ def save(self, filepath=None, setupProjectFile=True, template=False):
                 Graph.IO.Keys.Graph: self.getNonDefaultAttributes()
             }
         else:
+            self.header["template"] = False
```

Comment:
`self.header["template"] = template` out of the if/else

Reply:
Done